eq-author/src/components/CollapsibleToggled/index.js

```diff
@@ -67,4 +68,7 @@ CollapsibleToggled.propTypes = {
    * The content to show when the collapsible is open.
    */
   children: PropTypes.node.isRequired,
+  questionnaireId: PropTypes.string.isRequired,
+  shortName: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
```

Comment:
Could you please add a few lines to document what these props are (see above for how they should be formatted)? These filter through to Storybook to populate the `description` column in the prop table.

GitHub also wouldn't let me highlight lines 59 - 62. These should also be updated if you decide to implement my suggestion to update the name of the `defaultOpen` prop.